src/cljs-lib/src/calva/pprint/printer.cljs

```diff
@@ -1,43 +1,12 @@
 (ns calva.pprint.printer
-  (:require [cljs.test :refer [is testing]]
-            [zprint.core :refer [zprint-str]]
+  (:require [zprint.core :refer [zprint-str]]
             [calva.js-utils :refer [jsify cljify]]
             [clojure.string]))
 
 (defn pretty-print
   "Parses the string `s` as EDN and returns it pretty printed as a string.
    Accepts that s is an EDN form already, and skips the parsing, if so.
    Formats the result to fit the width `w`."
-  {:test (fn []
-           (letfn [(pretty-line-of [n s opts]
-                     (as-> `[~@(repeat n s)] x
-                       (pretty-print x opts)
-                       (:value x)
-                       (clojure.string/split x #"\n")
-                       (take 1 x)))]
-             (let [deep [[[[[[[[[[[[[[[[[{:foo [:bar]}]]]]]]]]]]]]]]]]]
-                   shallow [[:foo]]]
-               (testing "String input"
-                 (is (= "[[[:foo]]]"
-                        (:value (pretty-print "[    [ [:foo
-                      ]]        ]" nil)))))
-               (testing "Valid and invalid EDN"
-                 (is (= "[1]"
-                        (:value (pretty-print "[   1]" nil))))
-                 (is (= "[  1"
-                        (:value (pretty-print "[  1" nil)))))
-               (testing "Default printing options" ; zprint default width 80
-                 (let [width (apply count (pretty-line-of 25 "foo" nil))]
-                   (is (> width 70))
-                   (is (<= width 80)))
-                 (is (not (re-find #"#" (:value (pretty-print deep nil))))))
-               (testing "Settings"
-                 (let [width (apply count (pretty-line-of 25 "foo" {:width 40}))]
-                   (is (> width 30))
-                   (is (<= width 40)))
-                 (let [width (apply count (pretty-line-of 25 "foo" {:max-length 2}))]
-                   (is (< width 20)))
-                 (is (re-find #"#" (:value (pretty-print shallow {:max-depth 1}))))))))}
```

Comment:
This was moved into the test directory in conjunction with removing the ns-regexp from the shadow-cljs :test target, so that an error wasn't thrown because "vscode" couldn't be found, due to lsp.cljs now living in the `cljs-lib/src/calva` directory.

Reply:
All cljs tests should go in the test directory now.

Reply:
I don't know/remember why that one test living in the src file caused an issue, but there are other tests in other source files that don't, so I left those, and they're still run by the test script.